README.md

```diff
@@ -3,47 +3,60 @@
 # dockstore-load-tester
 Dockstore Load/Performance/Stress Tester
 
-## Running the tests
+### Requirements
 
-### Setup
+You need Java and Maven installed.
```

Comment:
Looks like this still uses/has a .travis.yml, should be circle
Also Java 8 should be 11 https://github.com/dockstore/dockstore-load-tester/blob/master/pom.xml#L10
Can be separate ticket

Reply:
Added CircleCI and removed Travis

098a684582f29d5989d9da812a4bd0378f748033